envbuilder.go

```diff
@@ -668,6 +675,20 @@ func Run(ctx context.Context, options Options) error {
 	}
 	_ = file.Close()
 
+	var exportEnvFile *os.File
+	if options.ExportEnvFile != "" && !skippedRebuild {
```

Comment:
A mention as to why we don't want to do this on rebuild would be helpful. Essentially your GitHub comments!